src/main/kotlin/org/rust/lang/core/psi/ext/RsDocAndAttributeOwner.kt

```diff
@@ -94,17 +99,21 @@ class QueryAttributes(
             .singleOrNull()
 
     val langAttribute: String?
-        get() = getStringAttribute("lang")
+        get() = getStringAttributes("lang").firstOrNull()
 
-    val deriveAttribute: RsMetaItem?
-        get() = attrByName("derive")
+    val deriveAttributes: Sequence<RsMetaItem>
+        get() = attrsByName("derive")
 
-    private fun getStringAttribute(attributeName: String): String? = attrByName(attributeName)?.value
+    // `#[attributeName = "Xxx"]`
+    private fun getStringAttributes(attributeName: String): Sequence<String?> = attrsByName(attributeName).map { it.value }
 
     val metaItems: Sequence<RsMetaItem>
         get() = attributes.mapNotNull { it.metaItem }
 
-    private fun attrByName(name: String) = metaItems.find { it.referenceName == name }
+    /**
+     * Get a sequence of all attributes named [name]
+     */
+    private fun attrsByName(name: String) = metaItems.filter { it.referenceName == name }
```

Comment:
Let's add return type for this function to have explicit returning type for all functions